intranet/apps/users/models.py

```diff
@@ -308,6 +309,29 @@ def tj_email(self):
 
         return "{}@{}".format(self.username, domain)
 
+    @property
+    def non_tj_email(self):
+        """
+        Returns the user's first non-TJ email found.
+
+        If a user has a primary email set and it is not their TJ email,
+        use that. Otherwise, use the first email found that is not their
+        TJ email.
+
+        Returns:
+            The first non-TJ email found for a user
+
+        """
+        tj_email = self.tj_email
+        if self.primary_email and self.primary_email != tj_email:
+            return self.primary_email
+
+        for email in self.emails.all():
```

Comment:
Perhaps let the DB handle this by replacing this for loop and the final `return None` with:
```python
return self.emails.exclude(address__iexact=tj_email).first()
```